### DISCUSSION TITLE
Upgrade question

### DISCUSSION THREAD

**Discussion by BrubiSQL** [trimble-oss/dba-dash#170] [Q&A] [answered]
Hi, it's me again :D 

Question.
If i have a version v2.14.0.0, can i skip v2.14.1 and go directly on to the version v2.15.0.0? 
Will i get all database upgrades? Or is needed to go step by step?

B.

**Comment by DavidWiseman** [comment, marked as answer]
Yes - you can go straight to the latest version.  The app uses an SSDT project for the database schema which builds a dacpac file.  The dacpac deployment works by creating a diff on the fly and working out what changes to apply.

**Reply by BrubiSQL** [reply]
Awsome! Thank you for the answer! :)
